src/js/components/TextInput/stories/textinput.stories.js

```diff
@@ -1,35 +1,31 @@
-import React, { Component, useState } from 'react';
+import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { Hide, View } from 'grommet-icons';
 
 import { Box, Grommet, TextInput, Button } from 'grommet';
 import { grommet } from 'grommet/themes';
 import { deepMerge } from 'grommet/utils';
 
-class SimpleTextInput extends Component {
-  state = { value: '' };
+const SimpleTextInput = () => {
+  const [value, setValue] = React.useState('');
 
-  ref = React.createRef();
+  const onChange = event => setValue(event.target.value);
 
-  onChange = event => this.setState({ value: event.target.value });
-
-  render() {
-    const { value } = this.state;
-    return (
-      <Grommet full theme={grommet}>
-        <Box fill align="center" justify="start" pad="large">
-          <Box width="medium">
-            <TextInput ref={this.ref} value={value} onChange={this.onChange} />
-          </Box>
+  return (
+    <Grommet full theme={grommet}>
+      <Box fill align="center" justify="start" pad="large">
+        <Box width="medium">
+          <TextInput value={value} onChange={onChange} />
         </Box>
-      </Grommet>
-    );
-  }
-}
+      </Box>
+    </Grommet>
+  );
+};
 
 const PasswordInput = ({ value, ...rest }) => {
-  const [inputValue, setValue] = useState(value);
-  const [reveal, setReveal] = useState(false);
+  const [inputValue, setValue] = React.useState(value);
```

Comment:
I'm seeing this console error when typing password
![image](https://user-images.githubusercontent.com/6320236/64025468-fd7ce900-caf9-11e9-965d-ba1c45560b54.png)